source/style-guide/markup/format/inline.txt

```diff
@@ -0,0 +1,110 @@
+=============
+Inline Markup
+=============
+
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Basic Markup
+------------
+
+- To **strongly emphasize** or **bold** text, use two asterisks on
```

Comment:
This should crosslink to the "Text Formatting" and "Titles and Headings" page to explain when to use them.